iron-collapse.html

```diff
@@ -128,6 +128,14 @@
         type: Boolean
       },
 
+      /**
+       * stores the desired size of the collapse body
```

Comment:
nit: can you capitalize first letter and end the sentence with a `.` ? (this will show in the docs on elements.polymer-project.org/elements/iron-collapse)